components/useCases/UseCases.js

```diff
@@ -0,0 +1,5 @@
+const UseCases = () => {
+  return <div>hello usecases</div>;
```

Comment:
instead of dummy text, add something like: `Usecases page in under development...`